### DISCUSSION TITLE
Save image seed in filename prefix

### DISCUSSION THREAD

**Discussion by ggenovez** [WASasquatch/was-node-suite-comfyui#425] [General]
hi all,

some of the save images allow you to save checkpoints, seeds, and other gerated data as part of the file name.

Since we already have a token for date as part of the path ([time(%Y-%m-%d)]) do we have something similar to save the checkpoint, seed, etc?

**Comment by WASasquatch** [comment]
There are certain nodes that can take S&R tags, but I think support for nodes like Image Save have to be manually added to it. To that effect, I've made some nodes that spit out the filename of models and stuff that can be used to build a filename.
